node/lifecycle_test.go

```diff
@@ -414,8 +415,51 @@ func testCreateStartDeleteScenario(ctx context.Context, t *testing.T, s *system)
 	}
 }
 
-func newPod() *corev1.Pod {
-	return &corev1.Pod{
+func BenchmarkCreatePods(b *testing.B) {
```

Comment:
What is the goal of adding a benchmark?

Reply:
It's mostly for memory allocations, and not for throughput. When you pass `-benchmem`, you can at least find out how many allocs are required per pod?

Reply:
Right, but what's the plan here? Is it to keep track of the allocations and signal when we increase/decrease number of allocs? I'm fine with any tests and benchmarks but I'd like to understand what the project will be gaining from this being introduced.

Reply:
The discussion I've been having with @cpuguy83 is that we should set a scalability target. The intent of this test is to help us verify we can hit that scalability number. The one we decided on was 100k pods @ some known amount of memory. The purpose of this benchmark is to help us identify that number. 

Prior to any large changes, people should post before / after numbers and ensure we don't breach our #.

Reply:
That's enough for me, thanks. Now, I wonder if the fact the test uses a fake client and an _empty_ control-plane has any impact - I think it does.